### PR TITLE
add readme update for sysdig-agent

### PR DESCRIPTION
**Pull request by bob-bins**

### CODE REVIEW

File: datastores/as_kubernetes_pods/manifests/elasticsearch/README.md [hunk 1]
```diff
@@ -24,6 +24,28 @@ The following steps only highlight steps specific to the secure Elasticsearch up
 
         kubectl -n $NAMESPACE create secret generic sg-config-files --from-file=datastores/as_kubernetes_pods/manifests/elasticsearch/sgconfig
 
+5. If you are running `sysdig-agent`, ensure this section exists in the `sysdig-agent` `configmap`:
```
Comment (bjhaid) [inline datastores/as_kubernetes_pods/manifests/elasticsearch/README.md]: is there a situation where the user will not be running `sysdig-agent`?

Comment (bob-bins) [reply datastores/as_kubernetes_pods/manifests/elasticsearch/README.md]: I should change that to be more specific. What I mean is if they are running sysdig-agent in $NAMESPACE, which to my understanding, most customers won't do.

File: datastores/as_kubernetes_pods/manifests/elasticsearch/README.md [hunk 2]
```diff
@@ -24,6 +24,28 @@ The following steps only highlight steps specific to the secure Elasticsearch up
...
+                  comm: java
+                conf:
+                  url: https://sysdigcloud-elasticsearch:9200
+                  username: readonly
+                  password: <SG_READONLY_PASSWORD from step 3>
+                  ssl_verify: false
+
+    And restart the agent if necessary to apply the new configs.
+
+        kubectl -n $NAMESPACE delete po -l app=sysdig-agent
```
Comment (mbreitung) [inline datastores/as_kubernetes_pods/manifests/elasticsearch/README.md]: please spell out `pod`

File: datastores/as_kubernetes_pods/manifests/elasticsearch/README.md [hunk 3]
```diff
@@ -24,6 +24,28 @@ The following steps only highlight steps specific to the secure Elasticsearch up
 
         kubectl -n $NAMESPACE create secret generic sg-config-files --from-file=datastores/as_kubernetes_pods/manifests/elasticsearch/sgconfig
 
+5. If you are running `sysdig-agent` in `$NAMESPACE` to monitor your Sysdig installation, ensure this section exists in the `sysdig-agent` `configmap`:
```
Comment (mbreitung) [inline datastores/as_kubernetes_pods/manifests/elasticsearch/README.md]: It does not matter which namespace the agent is running for this to work.

Comment (bob-bins) [reply datastores/as_kubernetes_pods/manifests/elasticsearch/README.md]: ah ok!

File: datastores/as_kubernetes_pods/manifests/elasticsearch/README.md [hunk 4]
```diff
@@ -24,7 +24,7 @@ The following steps only highlight steps specific to the secure Elasticsearch up
 
         kubectl -n $NAMESPACE create secret generic sg-config-files --from-file=datastores/as_kubernetes_pods/manifests/elasticsearch/sgconfig
 
-5. If you are running `sysdig-agent` to monitor your Elasticsearch cluster, ensure this section exists in the `sysdig-agent` `configmap`:
+5. If you are running `sysdig-agent` to monitor your Elasticsearch cluster, ensure the `sysdig-agent` `configmap` has the following Elasticsearch configuration in the `data.dragent.yaml.app_checks` section:
```
Comment (bjhaid) [inline datastores/as_kubernetes_pods/manifests/elasticsearch/README.md]: sorry I should have been more clear with my question, are there none `sysdig-agent` users who would read this documentation? If no, my suggested nit is:

```
If you are monitoring Elasticsearch cluster with `sysdig-agent`, ensure ....
```

File: datastores/as_kubernetes_pods/manifests/elasticsearch/README.md [hunk 5]
```diff
@@ -24,6 +24,28 @@ The following steps only highlight steps specific to the secure Elasticsearch up
 
         kubectl -n $NAMESPACE create secret generic sg-config-files --from-file=datastores/as_kubernetes_pods/manifests/elasticsearch/sgconfig
 
+5. If you are monitoring the Elasticsearch cluster with `sysdig-agent`, ensure the `sysdig-agent` `configmap` has the following Elasticsearch configuration in the `data.dragent.yaml.app_checks` section:
```
Comment (janoe) [inline datastores/as_kubernetes_pods/manifests/elasticsearch/README.md]: Maybe make sense to change the agent deployment adding the "##If enabling auth" part like in the rest of the backend deployments and document this in the readme?

Comment (bob-bins) [reply datastores/as_kubernetes_pods/manifests/elasticsearch/README.md]: I added that here: https://github.com/draios/sysdigcloud-kubernetes/pull/206

**Pull request merged**